tutorials/axios/axios-call.md

```diff
@@ -0,0 +1,10 @@
+To use axios in any of your projects you must first import it by assigning it to a variable, which is usually called axios, though you are free to name it anything else.
+`const axios = require('axios')`
```

Comment:
I'd add a newline here and use triple backticks to make this a full code sample rather than an inline code sample